utils/src/fuse.rs

```diff
@@ -252,12 +252,25 @@ fn fuse_kern_mount(
     subtype: &str,
     flags: MsFlags,
 ) -> io::Result<File> {
-    let file = OpenOptions::new()
+    let file = match OpenOptions::new()
         .create(false)
         .read(true)
         .write(true)
-        .open(FUSE_DEVICE)?;
-    let meta = mountpoint.metadata()?;
+        .open(FUSE_DEVICE)
+    {
+        Ok(v) => v,
+        Err(e) => {
+            error!("FUSE failed to open. {}", e);
+            return Err(e);
+        }
+    };
+    let meta = match mountpoint.metadata() {
```

Comment:
Um. I can't understand why such an update is causing `rustc` crashed.
But I suggest to remove all the existed target directories like `target-fusedev` and `target-virtiofs` from workspace. Then try to re-compile nydus by executing `make` from project home direcotry.
Sometimes, rustc is not stable. :-(

Reply:
I built my patch in two different environment. rustc crashes in both of them.